acceptance/extender_test.go

```diff
@@ -39,6 +39,11 @@ var (
 	extendTest           *PhaseTest
 )
 
+const (
+	msgPullingRemoteImage          = "Did not find cache key, pulling remote image"
+	msgErrRetrievingImageFromCache = "Error while retrieving image from cache: oci"
```

Comment:
i see --   ideally imo these constants would be used in the (production) code that generates the logs, and then consumed from the tests,  so that it's clear that those log lines are relied on someplace else outside the immediate domain,  and the tests would update effortlessly with the constants.

Reply:
Ohhhh I see... those log lines are emitted by kaniko :/